cloud/docker/docker.py

```diff
@@ -725,16 +725,18 @@ def has_changed(self):
         return False
 
     def get_inspect_image(self):
+        return self.client.inspect_image(self.module.params.get('image'))
+
+    def get_image_repo_tags(self):
```

Comment:
`get_inspect_image` is essentially `docker inspect image-name`, it does not have `RepoTags` key.

`self.client.inspect_image` works without a tag, but fails if image does not exist locally, I'm pushing fix right now.